server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -317,7 +317,7 @@ function makePlot(id, inData, type) {
  * stores it on a sessionStorage variable
  */
 function getMaster() {
-  $.getJSON('/rest/master', function(data) {
+  return $.getJSON('/rest/master', function(data) {
```

Comment:
I'm wondering if these functions which save endpoint data into the session can be further simplified, with something like:

```javascript
function saveToSession(endpoint) {
  return $.getJSON(endpoint, function(data) { sessionStorage[endpoint] = JSON.stringify(data); } );
}

// the following are now simple enough to inline
function getMaster() { return saveToSession('/rest/master'); }
function getZk() { return saveToSession('/rest/zk'); }
```